stdlib/public/Concurrency/AsyncThrowingStream.swift

```diff
@@ -12,92 +12,230 @@
+      ///
+      /// This value reprsents the successful enqueueing of an element, whether
+      /// the stream buffers the element or delivers it immediately to a pending
+      /// call to `next()`. The associated value `remaining` indicates the
+      /// number of remaining slots in the buffer at the point in time of
+      /// yielding.
+      ///
+      /// > Note: Acting on the remaining count is valid only when calls to
+      /// yield are mutually exclusive.
       case enqueued(remaining: Int)
       
-      /// Yielding resulted in not buffering an element because the
-      /// buffer was full. The element is the dropped value.
+      /// The stream didn't enqueue the element due to a full buffer.
+      ///
+      /// The associated element for this case is the element that the stream
+      /// dropped.
       case dropped(Element)
       
-      /// Indication that the continuation was yielded when the
-      /// stream was already in a terminal state: either by cancel or
-      /// by finishing.
+      /// The stream didn't enqueue the element because the stream was in a
+      /// terminal state.
+      ///
+      /// This indicates the stream terminated prior to calling `yield`, either
+      /// because the stream finished normally or through cancellation, or
+      /// threw an error
       case terminated
     }
     
     /// A strategy that handles exhaustion of a buffer’s capacity.
     public enum BufferingPolicy {
+
+      /// Continue to add to the buffer, treating its capacity as infinite.
       case unbounded
       
       /// When the buffer is full, discard the newly received element.
-      /// This enforces keeping the specified amount of oldest values.
+      ///
+      /// This strategy enforces keeping the specified amount of oldest values.
       case bufferingOldest(Int)
       
       /// When the buffer is full, discard the oldest element in the buffer.
-      /// This enforces keeping the specified amount of newest values.
+      ///
+      /// This strategy enforces keeping the specified amount of newest values.
       case bufferingNewest(Int)
     }
 
     let storage: _Storage
 
     /// Resume the task awaiting the next iteration point by having it return
-    /// normally from its suspension point or buffer the value if no awaiting
-    /// next iteration is active.
+    /// nomally from its suspension point with a given element.
     ///
     /// - Parameter value: The value to yield from the continuation.
+    /// - Returns: A `YieldResult` indicating the success or failure of the
+    ///   yield operation.
+    ///
+    /// If nothing is awaiting the next value, the method attempts to buffer the
+    /// result's element.
     ///
     /// This can be called more than once and returns to the caller immediately
-    /// without blocking for any awaiting consumption from the iteration.
+    /// without blocking for any awaiting consuption from the iteration.
```

Comment:
consumption meaning the consuming of values. this is a spelling mistake?

Reply:
No, this appears to be me forgetting how to type. Or maybe deleting a character without first knowing where my cursor is located. I'll revert this change on the next pass. Sorry for the mess.